src/ocean/util/app/ext/ReopenableFilesExt.d

```diff
@@ -70,14 +94,54 @@ public class ReopenableFilesExt : IApplicationExtension, ISignalExtExtension
 
     ***************************************************************************/
 
-    public this ( SignalExt signal_ext = null, int reopen_signal = SIGHUP )
+    public this ( SignalExt signal_ext = null,
+        int reopen_signal = SIGHUP )
```

Comment:
Why the change indent ?

Reply:
Since this is internal-only, should it be deprecated ?

Reply:
Interesting thing that this is not internal only - once could create their completely separate application hierarchy and use this. However, I will deprecate this, good catch :+1:

Indentation was changed via copy/pasting from other constructor. Will fix.